_coffee/application.coffee

```diff
@@ -134,6 +139,15 @@ categories =
 53143469 # ios-demos
 54135763 # accessibilitools
 50032733 # novoda.github.io
+2168934  # public-mvn-repo
+61706237 # bonfire-firebase-sample
+73312159 # dashboards
+78538262 # eslint-config-novoda
```

Comment:
Here is like all the repos collected. I just followed the pattern. 😄 
This part is commented out, so does not have any effect on the website. 

It is not really scalable to write down every one of them.